src/components/QuranReader/types.ts

```diff
@@ -28,4 +28,27 @@ export enum QuranFont {
   QPCHafs = 'qpc_uthmani_hafs',
 }
 
+export enum MushafLine {
+  FifteenLines = '15_lines',
+  SixteenLines = '16_lines',
+}
+
+export enum Mushaf {
+  QCFV2 = 1,
+  QCFV1 = 2,
+  Indopak = 3,
+  UthmaniHafs = 4,
+  KFGQPCHAFS = 5,
+  Indopak15Lines = 6,
+  Indopak16Lines = 7,
+}
+
+export const QuranFontMushaf: Record<QuranFont, Mushaf> = {
+  [QuranFont.MadaniV1]: Mushaf.QCFV1,
+  [QuranFont.MadaniV2]: Mushaf.QCFV2,
+  [QuranFont.Uthmani]: Mushaf.UthmaniHafs,
+  [QuranFont.IndoPak]: Mushaf.Indopak,
+  [QuranFont.QPCHafs]: Mushaf.KFGQPCHAFS,
+};
+
```

Comment:
Not related to this PR, but have a feeling that this file needs to be moved out of components to the types directory.